docker/stan-wasm-server/logic/compilationJobMgmt.py

```diff
@@ -0,0 +1,105 @@
+from uuid import uuid4
+from string import hexdigits
+import os
+from fastapi import HTTPException
+from .definitions import CompilationStatus
+
+from .fileValidation.compilationFiles import download_filename_is_valid, write_stan_code_file
+
+### QUERY: It'd be nice to avoid taking a dependency on HTTPException here,
+# but the best I can think of to do it is to define a bunch of our own exceptions,
+# which would then have to be caught at the routing-functions level, which is
+# not a useful encapsulation & will prompt a lot of boilerplate code.
+# Thoughts?
```

Comment:
It seems like we could use https://fastapi.tiangolo.com/tutorial/handling-errors/#install-custom-exception-handlers to define something up in main that re-interpreted our custom exceptions. I agree this would be nicer.